models/sales/intermediate/int_hubspot__deals_enhanced.sql

```diff
@@ -5,6 +5,20 @@ with deals as (
     select *
     from {{ var('deal') }}
 
+), merged_deals as (
+
+    select *
+    from {{ var('merged_deal')}}
+
+), aggregate_merged_deals as (
+
+    select
+        deal_id,
+        {{ fivetran_utils.array_agg("merged_deal_id") }} as merged_deal_ids
+    
+    from {{ var('merged_deal')}}
+    group by 1
```

Comment:
Additionally, following the conversation from the source PR we may want to add a conditional to the merged deal components if we do decide to leverage a variable to enable/disable these.

Reply:
Included `hubspot_merged_deal_enabled` config